Core/src/main/java/com/paypal/android/core/graphql/fundingEligibility/FundingEligibilityResponse.kt

```diff
@@ -0,0 +1,37 @@
+package com.paypal.android.core.graphql.fundingEligibility
+
+import com.paypal.android.core.graphql.common.getJSONArrayOrNull
+import com.paypal.android.core.graphql.common.toStringsList
+import org.json.JSONObject
+
+data class FundingEligibilityResponse(
```

Comment:
Same here. It'll be better for the project structure if we break these out into their own files.